### PR TITLE
wp reduce serialisations and db calls

### PR DESCRIPTION
**Pull request by almogdepaz**
pull less blocks in cache warmup
split wp object to save on redundant wp to bytes
batch pull all ses blocks

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
Looks good

**Comment by almogdepaz** [comment]
there's one more chage i wanna do here before we merge
 to reduce more database calls

**Comment by almogdepaz** [comment]
> Looks good

had some more i could do,  ready.

File: src/types/weight_proof.py [hunk 1]
```diff
@@ -80,6 +80,13 @@ class SubEpochSegments(Streamable):
     challenge_segments: List[SubEpochChallengeSegment]
 
 
+@dataclass(frozen=True)
+@streamable
+# this is used only for serialization to database
+class RecentChainData(Streamable):
```
Comment (mariano54) [inline src/types/weight_proof.py:86]: If we are changing the DB format won't that break old nodes that have the old format?

Comment (almogdepaz) [reply src/types/weight_proof.py:86]: bad commet ill fix, used for serialization to the multi process

Comment (almogdepaz) [reply src/types/weight_proof.py:86]: dident change any of the dbs structure

**Review by mariano54 — DISMISSED** [review]

**Comment by almogdepaz** [comment]
opened new pr for easier merge

**Pull request closed without merging**